cookbooks/aws-parallelcluster-install/attributes/default.rb

```diff
@@ -4,3 +4,41 @@
 default['cluster']['sysctl']['ipv4']['gc_thresh1'] = 0
 default['cluster']['sysctl']['ipv4']['gc_thresh2'] = 15_360
 default['cluster']['sysctl']['ipv4']['gc_thresh3'] = 16_384
+
+# Lustre defaults (for CentOS >=7.7 and Ubuntu)
+default['cluster']['lustre']['public_key'] = value_for_platform(
```

Comment:
`value_for_platform` is a hidden if and therefore it should be moved to the resource, here and below.